### ISSUE TITLE
XML result not recognised as XML when pasting it at the input

### ISSUE THREAD

**Issue by pavelsaman** [firecamp-io/firecamp#50]
1. problem

When converting a valid JSON to XML, then copying the XML result and pasting it back at the input causes the XML to be unrecognised as XML input.

2. steps to reproduce

Copy a valid JSON:

```
{
   "a": 5,
   "b": [
      1,
      2,
      3
   ],
   "C": "bla",
   "d": {
      "e": 1.25487
   }
}
```
=> convert it into XML, it will give you the following XML:

```
<a>5</a>
<b>1</b>
<b>2</b>
<b>3</b>
<C>bla</C>
<d>
    <e>1.25487</e>
</d>
```
=> copying and pasting the XML at the input => it's not recognised as XML:
![image](https://user-images.githubusercontent.com/24702416/73589312-fad15600-44d4-11ea-8261-749dbb76eed2.png)

3. desired behaviour

It's possible to convert back and forth using just the JSON, XML, YAML utility within firecamp with no need to manually add/remove tags etc. In this case, the `<root></root>` tag might be missing, since converting the following XML

```
<root>
  <a>5</a>
  <b>1</b>
  <b>2</b>
  <b>3</b>
  <C>bla</C>
  <d>
    <e>1.25487</e>
  </d>
</root>
```

will yield the expected behaviour.

4. version

Snap installed firecamp v.0.9.2

**Comment by Nishchit14** [comment]
we heavily depend on the other libraries to convert data format internally. I need to check the possibility.

**Comment by pranavwani** [comment]
The issue is fixed and shipped with [v2.5.0](https://github.com/firecamp-io/firecamp/releases/tag/v2.5.0)
